modules/store/spec/types/action_group_creator.spec.ts

```diff
@@ -70,6 +70,17 @@ describe('createActionGroup', () => {
       });
     });
 
+    describe('events', () => {
+      it('should infer events defined as an empty object', () => {
```

Comment:
For the completeness, could you also add a test here to infer the actions with events?